### ISSUE TITLE
does Sphinx autosummary work with the MATLAB domain?

### ISSUE THREAD

**Issue by joeced** [sphinx-contrib/matlabdomain#37]
Originally reported by: **Lev Givon (Bitbucket: [lebedov](https://bitbucket.org/lebedov), GitHub: [lebedov](https://github.com/lebedov))**

----------------------------------------

I'm trying to generate docs for a toolbox of MATLAB functions, but it doesn't seem that autosummary can find the functions even though I have the base directory correctly set. See [this post](http://stackoverflow.com/questions/27515103/using-sphinx-autosummary-extension-with-matlab-domain) on StackOverflow for the original question.

----------------------------------------
- Bitbucket: https://bitbucket.org/bwanamarko/sphinxcontrib-matlabdomain/issue/32

**Comment by skycaptain** [comment]
Any news on this?

**Comment by joeced** [comment]
No, sorrry. No news. I have had very little time to work on this project.

**Comment by VincentRouanne** [comment]
First, thanks for the all the work already done on the matlab domain. 

As a new user of both sphinx and the matlab domain, I think it would be nice to mention this issue in the Readme, as it seems most sphinx tutorials and usecases assume that autosummary  is usable (or apidoc / autogen which seem to depend on autosummary).
